api/testworkflows/v1/types.go

```diff
@@ -228,18 +230,14 @@ type CronJobConfig struct {
 	Annotations map[string]string `json:"annotations,omitempty" expr:"template,template"`
 	// configuration to pass for the workflow
 	Config map[string]intstr.IntOrString `json:"config,omitempty" expr:"template,template"`
+	// Targets helps decide on which runner the execution is scheduled.
+	Target *commonv1.Target `json:"target,omitempty" expr:"include"`
 }
 
 type TestWorkflowTagSchema struct {
```

Comment:
not important: Probably worth to rename this `struct`, as it's probably `TestWorkflowExecutionConfig` now